### PR TITLE
cli updates 2

### PR DESCRIPTION
**Pull request by efriis**
- extras group
- readme
- another readme

### CODE REVIEW

File: libs/cli/pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,10 @@ ruff = "^0.1.3"
 
 [tool.poetry.group.typing.dependencies]
 
+[tool.poetry.extras]
+# For langserve
+serve = ["langserve"]
```
Comment (efriis) [inline libs/cli/pyproject.toml:35]: @eyurtsev here

File: templates/README.md [hunk 2]
```diff
@@ -11,13 +11,7 @@ Some other helpful docs:
...
 
 ```shell
-pip install -U langchain-cli
-```
-
-Then, install `langserve`:
-
-```shell
-pip install -U "langserve[all]"
+pip install -U langchain-cli[serve]
```
Comment (eyurtsev) [inline templates/README.md:14]: are `"` needed?
```suggestion
pip install -U "langchain-cli[serve]"
```

File: libs/cli/pyproject.toml (hunk 1 above)
Comment (eyurtsev) [reply libs/cli/pyproject.toml:35]: Why is the `[serve]` needed -- is this for use cases of the cli that don't involve serving?

Comment (efriis) [reply libs/cli/pyproject.toml:35]: Yeah in the future, might just want to install `langchain-cli[smith]` or other submodule deps. Goal is a unified cli.

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**